nucypher/characters/control/controllers.py

```diff
@@ -178,27 +71,60 @@ def __init__(self,
         # Internals
         self._transport = None
 
-        # Hard-wire the character's output flow to the Emitter
-        self._internal_controller = character_controller
-        self._internal_controller.emitter = self.emitter
+        self.interface = interface
+
+        def set_method(name):
```

Comment:
Needs follow up for deeper integration IMO - perhaps move to a method explaining the polymorphic approach and implicit routing implicitly occurring here.   I think this can be named `self.bind_interface()`